api/src/services/workspace/workspace.hooks.ts

```diff
@@ -0,0 +1,57 @@
+import { authenticate } from '@feathersjs/authentication'
+import { Forbidden } from '@feathersjs/errors'
+import { HookContext } from '@feathersjs/feathers'
+import {
+  validateData,
+  resolveData,
+  validateQuery,
+  resolveQuery,
+  resolveExternal,
+} from '@feathersjs/schema'
+import { USER_PROFILE } from '@locokit/definitions'
+import { UserResult } from '../auth/user/user.schema'
+import { workspaceQueryValidator, workspaceResolvers } from './workspace.resolver'
+import { workspaceDataValidator } from './workspace.schema'
+
+export const workspaceHooks = {
+  around: {
+    all: [resolveExternal(workspaceResolvers.dispatch), validateData(workspaceDataValidator)],
+  },
+  before: {
+    get: [authenticate('jwt', 'public')],
+    find: [
+      authenticate('jwt', 'public'),
```

Comment:
Finally you put authenticate in before ?

Reply:
Well, yes, 
If you pay a bit more attention, you'll see only `get` and `find` are `public` allowed.
All others (`create`/`patch`/`remove`) are `jwt` only.

So I can't put the `authenticate` hook in the `all` `around` as the `public` strategy would be allowed for all ?